spec/fixtures/files/xml/core/collectionobject1.xml

```diff
@@ -221,14 +221,14 @@
       </objectProductionPlaceGroup>
     </objectProductionPlaceGroupList>
     <referenceGroupList>
-      <referenceGroup>
```

Comment:
changed order of the referenceGroup elements due to mapper limitation re: order of multi-authority fields